fe/fe-core/src/main/java/com/starrocks/connector/ConnectorPartitionTraits.java

```diff
@@ -283,7 +283,8 @@ public Set<String> getUpdatedPartitionNames(List<BaseTableInfo> baseTables,
                     long basePartitionVersion = latestPartitionInfo.get(basePartitionName).getModifiedTime();
 
                     MaterializedView.BasePartitionInfo basePartitionInfo = versionEntry.getValue();
-                    if (basePartitionInfo == null || basePartitionVersion != basePartitionInfo.getVersion()) {
+                    if ((basePartitionInfo == null || basePartitionVersion != basePartitionInfo.getVersion())
+                            && basePartitionVersion != -1) {
```

Comment:
better to use  `basePartitionVersion >=0`

Reply:
add some comment like "basePartitionVersion less than 0 is not valid"

Reply:
done